### ISSUE TITLE
Unclear documentation about deprecated PayerInfo->getShippingAddress()

### ISSUE THREAD

**Issue by JoeyNK** [paypal/PayPal-PHP-SDK#738]
### Required Information

- PHP Version: 7
- PayPal-PHP-SDK Version: 1.0
- Debug ID(s):

### Issue Description

I updated my SDK and I saw a notification that PayerInfo->getShippingAddress() is deprecated with the following doc piece:  "[DEPRECATED] Use shipping address present in purchase unit or at root level of checkout Session."

Could someone please be more specific about the class (object) from which I can obtain the payer's shipping address after a payment has been executed?

**Comment by bluk** [comment]
Please see the [payment execute response](https://developer.paypal.com/docs/api/payments/#payment_execute_response) documentation sample on the right area. You should see the `shipping_address` object in an `item_list` object in the `transactions` list.

**Comment by truefusion** [comment]
@bluk Is that to say that the only time we should pull shipping address is only after we have made a sale? For my current needs I would have desire to obtain that information prior to making a sale (e.g. address confirmation purposes). Currently the response received by using the code from the [get payment example](http://paypal.github.io/PayPal-PHP-SDK/sample/doc/payments/GetPayment.html) contains shipping information retrievable via PayerInfo->getShippingAddress(). By implication of the deprecation I am to assume the shipping information will not be there once the deprecation phase passes. Yet, documentation and samples do not appear to be helpful in preparing for future API changes where shipping information is desired before making a sale. What would be the currently recommended way to obtain shipping information prior to making a sale?

**Comment by bluk** [comment]
You should be able to GET the payment before executing (but after approval) and see the shipping address in the `item_list` object in the `transactions` list (similar to how you would with the older method before).

If that isn't what you want, can you please open a new issue?

**Comment by JoeyNK** [comment]
@bluk Thanks for the reply. I eventually figured that out on the same day by examining the raw response from PayPal.
